src/azure-cli-core/azure/cli/core/translator/__init__.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=unused-import
+
+# action
+from ._decorators import cls_action_wrapper, cls_action_factory_wrapper
```

Comment:
In my opinion, decorators like these are too intrusive. They extract information from Azure CLI **dynamically** during runtime. 

If I were you, I would **statically** parse source code files like `_params.py` with regex or string operation.